firmware/quadruna/BMS/src/io/io_tractiveSystem.c

```diff
@@ -120,26 +120,26 @@ float io_tractiveSystem_getCurrentLowResolution()
         return NAN;
     }
 
-    // HSNBV-D06 Output 2 (+/- 300A):
+    // DHAB S/124 Output 2 (+/- 500A):
     //
     // +------------------+                +-------------+
-    // | HSNBV-D06        |---<1k>---+---| ADC Channel |
+    // | DHAB S/124       |---<33k>---+---| ADC Channel |
     // | Output 2 Voltage |            |   +-------------+
     // +------------------+            |
-    //                               <2.2k>
+    //                              <60.4k>
     //                                 |
     //                                ===
     //                                GND
     //
-    //                                                                 1
-    // Current = (HSNBV-D06 Output 1 Voltage - Offset Voltage) x ---------------
+    //                                                                  1
+    // Current = (DHAB S/124 Output 1 Voltage - Offset Voltage) x ---------------
     //                                                             Sensitivity
-    //                                              1k + 2.2k
-    // HSNBV-D06 Output 2 Voltage = ADC Voltage x --------------
-    //                                                 2.2k
+    //                                              33k + 60.4k
+    // DHAB S/124 Output 2 Voltage = ADC Voltage x --------------
+    //                                                 60.4k
     // Offset Voltage = 2.5V
```

Comment:
Should this be a flat offset of 2.558V? Sorry for nitpick

Reply:
Oh and same for output voltage 1

Reply:
2.5V is from the datasheet, updated the comment to make this clear